### PR TITLE
Jekoch/make media tests pass

### PR DESCRIPTION
**Pull request by Jessica-Koch**
update selected media tests - please let me know if I missed something critical

### CODE REVIEW

File: packages/teams-js/test/public/media.spec.ts [hunk 1]
```diff
@@ -442,58 +444,72 @@ describe('media', () => {
...
+          imageProps: { imageOutputFormats: [media.ImageOutputFormats.PDF] },
+          maxMediaCount: 6,
+        };
+        try {
+          await media.selectMedia(mediaInputs, (error: SdkError, attachments: media.Media[]) => {
+            mediaError = error;
+          });
+        } catch (err) {
+          expect(err).not.toBeNull();
+
```
Comment (aeunms) [inline packages/teams-js/test/public/media.spec.ts:508]: Nit: extra newline

File: packages/teams-js/test/public/media.spec.ts [hunk 2]
```diff
@@ -442,58 +444,72 @@ describe('media', () => {
...
-      // TODO: fix bug and include
-      it.skip('selectMedia calls with successful result for mediaType = 1 with imageOutputFormats', () => {
-        return mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(() => {
+      it('selectMedia calls with successful result for mediaType = 1 with imageOutputFormats', done => {
+        mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(() => {
           mobilePlatformMock.setClientSupportedSDKVersion(imageOutputFormatsAPISupportVersion);
-          let mediaAttachments: media.Media[], mediaError: SdkError;
           const mediaInputs: media.MediaInputs = {
-            mediaType: media.MediaType.Image,
+            mediaType: media.MediaType.VideoAndImage,
```
Comment (aeunms) [inline packages/teams-js/test/public/media.spec.ts:451]: Was it intentional to change Image into VideoAndImage for the merge? If so, does it not work if with Image?

File: packages/teams-js/test/public/media.spec.ts [hunk 3]
```diff
@@ -442,58 +444,72 @@ describe('media', () => {
...
+              id: callbackId,
+              args: [undefined, filesArray],
+            },
+          } as DOMMessageEvent);
         });
       });
 
+      it('selectMedia call for mediaType = 1 and imageOutputFormats in mediaAPISupportVersion of platform support fails', async () => {
+        await mobilePlatformMock.initializeWithContext(FrameContexts.task, HostClientType.android);
+
```
Comment (aeunms) [inline packages/teams-js/test/public/media.spec.ts:494]: Nit: extra newline

**Review by aeunms — COMMENTED** [review]
Beautiful. Thanks so much for doing this work and doing it so quickly! I left some nit comments and a question.

File: packages/teams-js/test/public/media.spec.ts (hunk 2 above)
Comment (Jessica-Koch) [reply packages/teams-js/test/public/media.spec.ts:451]: i think this was a mistake, good catch!

**Review by aeunms — APPROVED** [review]
:shipit:

**Pull request merged**